dist/index.js

```diff
@@ -356,7 +356,7 @@ function runAction() {
                 core.warning(`The value specified for the pipeline duration is larger than Github's allowed default. Pipeline ${executionGraphId} will run with a duration of ${pipelineDuration / 1000} seconds.`);
             }
             while (!Object.values(constants.EndStates).includes(executionGraph["status"])) {
-                core.info(`  » Pipeline is still in progress, will check again in 15s.`);
```

Comment:
Actually with the latest changes we should be printing the actual value we use in the sleep, so neither 15 or 30, but get the variable value in seconds and print that